### ISSUE TITLE
How can I improve val_mrcnn_mask_loss and mrcnn_mask_loss?

### ISSUE THREAD

**Issue by K-M-Ibrahim-Khalilullah** [matterport/Mask_RCNN#2351]
@waleedka 
I am training MaskRCNN with my own dataset. The loss values are look like this:
Epoch 1895/2000
200/200 [==============================] - 161s 804ms/step - loss: 0.5184 - rpn_class_loss: 0.0224 - rpn_bbox_loss: 0.1595 - mrcnn_class_loss: 0.0464 - mrcnn_bbox_loss: 0.0663 - mrcnn_mask_loss: 0.2238 - val_loss: 2.2940 - val_rpn_class_loss: 0.0479 - val_rpn_bbox_loss: 0.8811 - val_mrcnn_class_loss: 0.4146 - val_mrcnn_bbox_loss: 0.4171 - val_mrcnn_mask_loss: 0.5333
Epoch 1896/2000
113/200 [===============>..............] - ETA: 1:05 - loss: 0.4443 - rpn_class_loss: 0.0166 - rpn_bbox_loss: 0.1182 - mrcnn_class_loss: 0.0413 - mrcnn_bbox_loss: 0.0552 - mrcnn_mask_loss: 0.2129

My question is How can I improve  val_mrcnn_mask_loss and  mrcnn_mask_loss?

Thanks

**Comment by kimile599** [comment]
I got another problem, my loss has lowered to about .1 or 0.09, but my val_loss is not converging at all. Have you met the same situation?
